python/orion_finance_sdk_py/contracts.py

```
@@ -337,6 +409,72 @@ def is_system_idle(self) -> bool:
+        return _call_view(self.contract.functions.priceAdapterRegistry())
+
+    @property
+    def price_adapter_decimals(self) -> int:
+        """Fetch the price adapter decimals from OrionConfig."""
+        return _call_view(self.contract.functions.priceAdapterDecimals())
+
+
+class PriceAdapterRegistry(OrionSmartContract):
+    """PriceAdapterRegistry contract for point-in-time asset prices."""
+
+    def __init__(self, contract_address: str | None = None):
```

Ruff (0.15.21)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)